README.md

```diff
@@ -1,2 +1,10 @@
-# opendr_internal
+# OpenDR Toolkit
 OpenDR Internal Repository
+
+
+This is the internal repository of [OpenDR Project](opendr.eu) that contains the development version of OpenDR toolkit. Note that this version is not yet production ready, so this repository is not yet intended to be used for any purpose, since it might contain unstable code that undergoes heavy development. The first public version of OpenDR toolkit is scheduled to be released on Dec. 2021.
```

Comment:
The URL is not working, probably it should be changed to 
```suggestion
This is the internal repository of [OpenDR Project](https://opendr.eu) that contains the development version of OpenDR toolkit. Note that this version is not yet production ready, so this repository is not yet intended to be used for any purpose, since it might contain unstable code that undergoes heavy development. The first public version of OpenDR toolkit is scheduled to be released on Dec. 2021.
```

Side note, writing one sentence per line makes the review in the GitHub UI easier.

Reply:
Thanks! I fixed the url and split the sentences to one per line.